src/rosdistro/__init__.py

```diff
@@ -149,7 +149,7 @@ def get_cached_distribution(index, dist_name, cache=None, allow_lazy_load=False)
     return dist
 
 
-def get_distribution_cache(index, dist_name):
+def fetch_distribution_cache_file(index, dist_name):
```

Comment:
Since the function doesn't fetch just the file (which might be gzipped) how about renaming the new function to `get_distribution_cache_string`?

Reply:
Good idea, thank you!